veil-tests/src/compile_tests/fail/redact_display_enum_variant.stderr

```diff
@@ -0,0 +1,17 @@
+error: `#[redact(display)]` is invalid here
+ --> src/compile_tests/fail/redact_display_enum_variant.rs:5:5
+  |
+5 |     #[redact(variant, display)]
+  |     ^
+
+error: expected `#[redact(all, ...)]` or `#[redact(variant, ...)]`, or both as separate attributes
+  --> src/compile_tests/fail/redact_display_enum_variant.rs:11:5
+   |
+11 |     #[redact(display)]
+   |     ^
+
+error: `#[redact(display)]` is invalid here
+  --> src/compile_tests/fail/redact_display_enum_variant.rs:16:1
+   |
+16 | #[redact(all, variant, display)]
+   | ^
```

Comment:
It feels like these messages should be inverted

Reply:
Okay, so the error was actually correct, it was just a nonsense message, so I've tried to make it more descriptive